src/pages/myGeotab/guides/usingCustomTelematicsDevices.tsx

```diff
@@ -24,58 +26,72 @@ const deviceManagement: ReactNode = (
   <div className="paragraph">
     Geotab uses MyAdmin API to manage devices and uses Data Intake Gateway to
     intake device data.
-    <br></br>
+    <br />
     <img
       src={myAdminAndDIGFlow}
       className="usingCustomTelematicsDevices__deviceManagementIMG"
       alt="Navigate to MyAdmin introduction"
     />
-    <br></br>
+    <br />
     To manage devices, the following MyAdmin API calls are most frequently used:
     <ul>
       <li>
-        <a hrefLang="../api/reference/#Authenticate">Authenticate</a>{" "}
-        {/* TODO: Need to replace this with routing to Authenticate page*/} (an
-        example is given in{" "}
-        <a href="https://docs.google.com/document/d/1Mddnxc2qKBCNYvVu0-BXcyR-blPlHwa0Zun0mBzZt88/edit#heading=h.bub5istj4ydv">
+        <HashLink to="/myGeotab/apiReference/methods#AuthenticateAsync">
+          Authenticate
+        </HashLink>{" "}
+        (an example is given in{" "}
+        <a
+          href="https://docs.google.com/document/d/1Mddnxc2qKBCNYvVu0-BXcyR-blPlHwa0Zun0mBzZt88/edit#heading=h.bub5istj4ydv"
+          target="_blank"
+          rel="noreferrer"
+          aria-label="What is the Authenticate MyAdmin API method?"
+        >
           What is the Authenticate MyAdmin API method
         </a>
         )
       </li>
       <li>
-        <a hrefLang="../api/reference/#ProvisionDeviceToAccount_1">
+        <HashLink to="/myAdmin/apiReference/methods#ProvisionDeviceToAccount_1">
           ProvisionDeviceToAccount
-        </a>{" "}
-        {/* TODO: Need to replace this with routing to ProvisionDeviceToAccount page*/}{" "}
+        </HashLink>{" "}
         (an example is given in{" "}
-        <a href="https://docs.google.com/document/d/1Mddnxc2qKBCNYvVu0-BXcyR-blPlHwa0Zun0mBzZt88/edit#heading=h.eghr27v40pev">
+        <a
+          href="https://docs.google.com/document/d/1Mddnxc2qKBCNYvVu0-BXcyR-blPlHwa0Zun0mBzZt88/edit#heading=h.eghr27v40pev"
+          target="_blank"
+          rel="noreferrer"
+          aria-label="What are the ProvisionDevice/ProvisionDeviceToAccount MyAdmin API method"
+        >
           What are the ProvisionDevice/ProvisionDeviceToAccount MyAdmin API
           method
         </a>
         )
       </li>
       <li>
-        <a hrefLang="../api/reference/#ProvisionDevice_1">ProvisionDevice</a>{" "}
-        {/* TODO: Need to replace this with routing to ProvisionDevice_1 page*/}{" "}
+        <HashLink to="/myAdmin/apiReference/methods#ProvisionDevice_1">
```

Comment:
maybe add a TODO here to check back to see if this links properly when page is available & the component Id is the same